src/devicemanager/devicemanager.cpp

```diff
@@ -59,12 +61,16 @@ void DeviceManager::append(const LinkConfiguration& linkConf, const QString& dev
 void DeviceManager::startDetecting()
 {
     qCDebug(DEVICEMANAGER) << "Start protocol detector service.";
+    _broadcastTimer.start(1000);
+    _broadcastSocket.bind(QHostAddress::Any, 1048);
```

Comment:
Thy is this in device manager and no in protocoldetector ?
DeviceManager is just a class that manages the device creation from the links that are possible to connect, everything related to the discovery process should be in protocol detector

Reply:
My reasoning was that we don't need to detect the protocol. We know it is a Ping360. I can move it over, but I vote for renaming ProtocolDetector to DeviceDetector or something like that =]

Reply:
Indeed, We should rework and make it more abstract and clear what is the paper of each class.
Also, we should think more about the protocol detection side of things, allowing multiple protocols, different proprietary sensors and etc.